autogpt/commands/command.py

```diff
@@ -133,12 +138,17 @@ def import_commands(self, module_name: str) -> None:
 def command(
     name: str,
     description: str,
-    signature: str = "",
-    enabled: bool = True,
+    signature: str,
+    enabled: bool | Callable[[Config], bool] = True,
     disabled_reason: Optional[str] = None,
 ) -> Callable[..., Any]:
     """The command decorator is used to create Command objects from ordinary functions."""
 
+    # TODO: Remove this in favor of better command management
+    CFG = Config()
```

Comment:
This will be removed in a later refactoring. This PR aims to open up commands to be converted to actions. The removal of this CFG is out of scope for it